src/browser/components/MattermostView.jsx

```diff
@@ -44,8 +43,18 @@ const MattermostView = createReactClass({
     var self = this;
     var webview = findDOMNode(this.refs.webview);
 
+    ipcRenderer.on('protocol-deeplink', (event, lastUrl) => {
```

Comment:
You should handle which tab should load the deeplink. Otherwise, non-appropriate tabs may also open the link.